### DISCUSSION TITLE
Qingping CGG1-T Homekit/Thread Version

### DISCUSSION THREAD

**Discussion by mad-scientist** [pvvx/ZigbeeTLc#92] [Q&A]
Hi! I see that the CGG1-M version is currently in development. Any idea whether the CGG1-T (Bluetooth + Homekit/Thread) version will be supported as well?

Thanks!

**Comment by pvvx** [comment]
The repository header contains the response:
_Custom firmware for Zigbee 3.0 IoT devices on the_ **TLSR825x** _chip_

**Reply by johnclegg-nz** [reply]
Is the CGG1-T a different chipset ?

**Reply by mad-scientist** [reply]
I haven't opened the device ...

**Reply by pvvx** [reply]
> Is the CGG1-T a different chipset ?

![image](https://github.com/pvvx/ZigbeeTLc/assets/12629515/fb81aecf-8c52-4871-9b14-97c88b4b4fbd)

TLSR825x is only used in the M version.

Versions T and H usually use an nRF chip and have a markup of 2..3 times due to the Apple label.
